src/services/authentication/strategies/TSPStrategy.js

```diff
@@ -0,0 +1,115 @@
+const { AuthenticationBaseStrategy } = require('@feathersjs/authentication');
+const { NotAuthenticated } = require('@feathersjs/errors');
+const { omit } = require('lodash');
+//const { jose } = require('jose');
+
+class TSPStrategy extends AuthenticationBaseStrategy {
+
+	get configuration() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Block must not be padded by blank lines. (padded-blocks)](https://app.codacy.com/app/schul-cloud/schulcloud-server/pullRequest?prid=4491705)